interfaces/builtin/udisks2.go

```diff
@@ -0,0 +1,920 @@
+    bus=system
+    path=/org/freedesktop/UDisks2/drives/*
+    interface=org.freedesktop.UDisks2.Drive
+    member=SetConfiguration
+    peer=(label=###SLOT_SECURITY_TAGS###),
+
+dbus (send)
+    bus=system
+    path=/org/freedesktop/UDisks2/drives/*
+    interface=org.freedesktop.UDisks2.Drive
+    member=PowerOff
+    peer=(label=###SLOT_SECURITY_TAGS###),
+
+dbus (send)
+    bus=system
+    path=/org/freedesktop/UDisks2/drives/*
+    interface=org.freedesktop.DBus.Properties
+    member=Get
+    peer=(label=###SLOT_SECURITY_TAGS###),
+    
+dbus (receive)
+    bus=system
+    path=/org/freedesktop/UDisks2/drives/*
+    interface=org.freedesktop.DBus.Properties
+    member=PropertiesChanged
+    peer=(label=###SLOT_SECURITY_TAGS###),
+
+# Drive.Ata
+dbus (send)
+    bus=system
+    path=/org/freedesktop/UDisks2/drives/*
+    interface=org.freedesktop.UDisks2.Drive.Ata
+    member=Smart{Update,GetAttributes,SelftestStart,SelftestAbort,SetEnabled}
+    peer=(label=###SLOT_SECURITY_TAGS###),
+
+dbus (send)
+    bus=system
+    path=/org/freedesktop/UDisks2/drives/*
+    interface=org.freedesktop.UDisks2.Drive.Ata
+    member=Pm{GetState,Standby,Wakeup}
+    peer=(label=###SLOT_SECURITY_TAGS###),
+
+dbus (send)
+    bus=system
+    path=/org/freedesktop/UDisks2/drives/*
+    interface=org.freedesktop.UDisks2.Drive.Ata
+    member=SecurityEraseUnit
+    peer=(label=###SLOT_SECURITY_TAGS###),
+
+# UDisks2 MDRaid interface
+dbus (send)
+    bus=system
+    path=/org/freedesktop/UDisks2/mdraid/*
+    interface=org.freedesktop.UDisks2.MDRaid
+    member={Start,Stop}
+    peer=(label=###SLOT_SECURITY_TAGS###),
+    
+dbus (send)
+    bus=system
+    path=/org/freedesktop/UDisks2/mdraid/*
+    interface=org.freedesktop.UDisks2.MDRaid
+    member={Add,Remove}Device
+    peer=(label=###SLOT_SECURITY_TAGS###),
+
+dbus (send)
+    bus=system
+    path=/org/freedesktop/UDisks2/mdraid/*
+    interface=org.freedesktop.UDisks2.MDRaid
+    member=SetBitmapLocation
+    peer=(label=###SLOT_SECURITY_TAGS###),
+
+dbus (send)
+    bus=system
+    path=/org/freedesktop/UDisks2/mdraid/*
+    interface=org.freedesktop.UDisks2.MDRaid
+    member=RequestSyncAction
+    peer=(label=###SLOT_SECURITY_TAGS###),
+
+dbus (send)
+    bus=system
+    path=/org/freedesktop/UDisks2/mdraid/*
+    interface=org.freedesktop.DBus.Properties
+    member=Get
+    peer=(label=###SLOT_SECURITY_TAGS###),
+    
+dbus (receive)
+    bus=system
+    path=/org/freedesktop/UDisks2/mdraid/*
+    interface=org.freedesktop.DBus.Properties
+    member=PropertiesChanged
+    peer=(label=###SLOT_SECURITY_TAGS###),
+
+# Udisks2 Job interface
+dbus (send)
+    bus=system
+    path=/org/freedesktop/UDisks2/jobs/*
+    interface=org.freedesktop.UDisks2.Job
+    member=Cancel
+    peer=(label=###SLOT_SECURITY_TAGS###),
+
+dbus (receive)
+    bus=system
+    path=/org/freedesktop/UDisks2/jobs/*
+    interface=org.freedesktop.UDisks2.Job
+    member=Completed
+    peer=(label=###SLOT_SECURITY_TAGS###),
```

Comment:
The same of course goes for all these ConnectedPlug rules. Eg, use either a per dbus interface glob:
```
dbus (receive, send)
    bus=system
    path=/org/freedesktop/UDisks2/jobs/*
    interface=org.freedesktop.UDisks2.Job
    peer=(label=###SLOT_SECURITY_TAGS###),
```

or a big glob:
```
dbus (receive, send)
    bus=system
    path=/org/freedesktop/UDisks2/**
    interface=org.freedesktop.UDisks2.*
    peer=(label=###SLOT_SECURITY_TAGS###),
```